operator/v1/types_ingress.go

```diff
@@ -501,6 +579,18 @@ type AccessLogging struct {
 	//
 	// +optional
 	HttpLogFormat string `json:"httpLogFormat,omitempty"`
+
+	// httpCaptureHeaders defines HTTP headers that should be captured in
+	// access logs.  If this field is empty, no headers are captured.
+	//
+	// +optional
+	HTTPCaptureHeaders *IngressControllerCaptureHTTPHeaders `json:"httpCaptureHeaders,omitempty"`
```

Comment:
I don't see why this one should be a pointer. The slices can just be zero length and it does the same thing, right?

Reply:
Right, that would work.  Updated.